### PR TITLE
Improve mandatory fields visibility 

### PR DESCRIPTION
**Pull request by NileshPant1999**

### CODE REVIEW

File: src/app/shared/mandatory-field/mandatory-field.component.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.required-field {
+  font-size: 15px;
+  color: #f36;
+}
```
Comment (ashwin1111) [inline src/app/shared/mandatory-field/mandatory-field.component.scss:4]: new line

**Review by ashwin1111 — CHANGES_REQUESTED** [review]

File: src/app/qbo/settings/schedule/schedule.component.ts [hunk 2]
```diff
@@ -35,26 +35,23 @@ export class ScheduleComponent implements OnInit {
...
       that.isLoading = false;
-    }, (error) => {
+    }, () => {
       that.isLoading = false;
     });
   }
 
   submit() {
     const that = this;
-    if (that.form.valid) {
```
[the remark this answers is not in the document]
Comment (NileshPant1999) [reply src/app/qbo/settings/schedule/schedule.component.ts:45]: yes

File: src/app/qbo/settings/general-mappings/general-mappings.component.ts [hunk 3]
```diff
@@ -70,62 +59,73 @@ export class GeneralMappingsComponent implements OnInit {
...
     }
-    if (cccAccountId === null) {
-      this.cccAccountIsValid = true;
+
+    if (that.generalSettings.sync_fyle_to_qbo_payments) {
+      that.form.controls.billPaymentAccounts.setValidators(Validators.required);
     }
+  }
 
-    if (that.accountsPayableIsValid && that.bankAccountIsValid && that.cccAccountIsValid && that.vendorIsValid && that.billPaymentAccountIsValid) {
```
[the remark this answers is not in the document]
Comment (NileshPant1999) [reply src/app/qbo/settings/general-mappings/general-mappings.component.ts:95]: yes

File: src/app/qbo/settings/configuration/general-configuration/general-configuration.component.ts [hunk 4]
```diff
@@ -216,89 +215,85 @@ export class GeneralConfigurationComponent implements OnInit {
 
   save() {
     const that = this;
-    if (that.generalSettingsForm.valid) {
```
[the remark this answers is not in the document]
Comment (NileshPant1999) [reply src/app/qbo/settings/configuration/general-configuration/general-configuration.component.ts:219]: yes

**Review by ashwin1111 — APPROVED** [review]

**Review by Sravanksk — APPROVED** [review]

**Pull request merged**